<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>killPage</class>
 <widget class="QWidget" name="killPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>792</height>
   </rect>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>771</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>859</width>
      <height>769</height>
     </rect>
    </property>
    <widget class="QLabel" name="shotByLabel_6">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Käyttö</string>
     </property>
    </widget>
    <widget class="QComboBox" name="animalComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="usageComboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="usage2CheckBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Toinen Käyttö</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="locationLineEdit">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Karttaan merkitty paikannimi</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="usage2PortionSpinBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>180</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>25</number>
     </property>
    </widget>
    <widget class="QPushButton" name="saveShotPushButton">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>210</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Tallenna</string>
     </property>
    </widget>
    <widget class="QLabel" name="shotDateLabel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Kaatopäivä</string>
     </property>
    </widget>
    <widget class="QLabel" name="shotByLabel_4">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sukupuoli</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="killsKillsTableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>841</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="shotByLabel_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Paino (kg)</string>
     </property>
    </widget>
    <widget class="QLabel" name="shotByLabel_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Eläin</string>
     </property>
    </widget>
    <widget class="QComboBox" name="usage2ComboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="genderComboBox">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>80</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="shotByLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Kaataja</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="weightLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="editShotsPushButton">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>500</y>
       <width>130</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Muokkaa kaatoja...</string>
     </property>
    </widget>
    <widget class="QLabel" name="shotByLabel_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ikäryhmä</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>550</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Vuosi</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="shotByComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="shotDateEdit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="locale">
      <locale language="Finnish" country="Finland"/>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="date">
      <date>
       <year>2022</year>
       <month>11</month>
       <day>15</day>
      </date>
     </property>
    </widget>
    <widget class="QLabel" name="shotByLabel_7">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>110</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Lisätietoja</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="usagePortionSpinBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>130</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>25</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="additionalInfoTextEdit">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>130</y>
       <width>220</width>
       <height>70</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Tähän vieraan kaatajan nimi tai havaintoja eläimestä</string>
     </property>
     <property name="toolTipDuration">
      <number>10000</number>
     </property>
    </widget>
    <widget class="QTableWidget" name="shotLicenseTableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>580</y>
       <width>521</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="shotDateLabel_2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Paikka</string>
     </property>
    </widget>
    <widget class="QLabel" name="shotByLabel_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>550</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Käytetyt Luvat</string>
     </property>
    </widget>
    <widget class="QComboBox" name="licenseYearComboBox">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>550</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="shotByLabel_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Kaadot</string>
     </property>
    </widget>
    <widget class="QLabel" name="shotByLabel_17">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>160</y>
       <width>130</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Toisen käytön osuus</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ageGroupComboBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="shotByLabel_16">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>80</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Käytön osuus</string>
     </property>
    </widget>
    <widget class="QComboBox" name="sortKillsComboBox">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>240</y>
       <width>130</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>217</y>
       <width>130</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Lajittele</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeShotsPushButton">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>500</y>
       <width>130</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Poista kaato...</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
